Error:  [2016/5/31 11:04:02] ---- Log: 启动服务失败。
Exc.Message: Unable to connect to any of the specified MySQL hosts.Exc.Stack:    在 MySql.Data.MySqlClient.NativeDriver.Open()
   在 MySql.Data.MySqlClient.Driver.Open()
   在 MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   在 MySql.Data.MySqlClient.MySqlConnection.Open()
   在 DataBaseProvider.OrderDBProvider.GetAllFinishedSellOrders() 位置 f:\GitHub\SuperMiners\SuperMinersServerApplication\DataBaseProvider\OrderDBProvider.cs:行号 78
   在 SuperMinersServerApplication.Controller.OrderController.Init() 位置 f:\GitHub\SuperMiners\SuperMinersServerApplication\SuperMinersServerApplication\Controller\OrderController.cs:行号 37
   在 SuperMinersServerApplication.SuperMinersService.InitService() 位置 f:\GitHub\SuperMiners\SuperMinersServerApplication\SuperMinersServerApplication\SuperMinersService.cs:行号 74
-----------------------------------------------------

 datatables, Int32 startRecord, Int32 maxRecords, String srcTable,